javascripts/discourse/widgets/dc-topic-title.js.es6

```diff
@@ -27,15 +27,17 @@ export default createWidget("dc-topic-title", {
   buildTopicTags(attrs) {
     const tags = attrs.tags;
 
-    return tags.map(tag => h('a.dc-topic-tag', {
-      attributes: {
-        href: `/tag/${tag}`
-      }
-    }, tag));
+    if (tags) {
+      return tags.map(tag => h('a.dc-topic-tag', {
+        attributes: {
+          href: `/tag/${tag}`
+        }
+      }, tag));
+    }
   },
 
   buildTopicInfo(attrs) {
-    return [this.buildCategory(attrs), this.buildTopicTags(attrs)];
+    return [this.buildCategory(attrs), this.buildTopicTags(attrs)].filter(Boolean);
```

Comment:
@arathjz what's the idea behind this?

Reply:
Now that I think about it does it is not needed at all. It was just in case any of the two functions returned null or undefined